src/activities.c

```diff
@@ -1462,7 +1462,30 @@ godot_variant activity_manager_register_command(godot_object *p_instance, Librar
 
     return result_variant;
 }
+godot_variant activity_manager_register_steam(godot_object *p_instance, Library *p_lib,
+                                                ActivityManager *p_activity_manager,
+                                                int p_num_args, godot_variant **p_args)
+{
+    godot_variant result_variant;
+
+    if (p_num_args == 1) // Command
+    {
+        godot_int command_int = p_lib->api->godot_variant_as_int(p_args[0]);
+        godot_char_int command_char_int = p_lib->api->godot_string_to_int(&command_int);
 
+        const int *steamid = p_lib->api->godot_char_string_get_data(&command_char_int);
```

Comment:
There is no such thing as a godot_char_int, and the steamid does not need a pointer. You can delete lines 1474 and 1476:
```c
godot_int steamid = p_lib->api->godot_variant_as_int(p_args[0]);
```